File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
       Map seen = new HashMap();
       for (int i = 0; i < nums.length; i++) {
        if (seen.containsKey(target - nums[i])) {
            return new int[] {i, (int)seen.get(target - nums[i])};
        }
        else {
            seen.put(nums[i], i);
        }
       }
       return new int[] {0, 0};
    }
}

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        seenNum = {}
        for i in range(len(nums)):
            if target - nums[i] in seenNum:
                return [seenNum[target - nums[i]], i]
            seenNum[nums[i]] = i
        return []
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        